<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLModelDisplayNodeWidget</class>
 <widget class="QWidget" name="qMRMLModelDisplayNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Display Node</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="qMRMLDisplayNodeWidget" name="MRMLDisplayNodeWidget"/>
   </item>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="ScalarsGroupBox">
     <property name="title">
      <string>Scalars</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="ScalarsVisibilityLabel">
        <property name="text">
         <string>Visible:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="ScalarsVisibilityCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ActiveScalarLabel">
        <property name="text">
         <string>Active Scalar:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="ctkVTKDataSetArrayComboBox" name="ActiveScalarComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ScalarsColorTableLabel">
        <property name="text">
         <string>Color Table:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="qMRMLColorTableComboBox" name="ScalarsColorNodeComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ScalarRangeModeLabel">
        <property name="text">
         <string>Scalar Range Mode:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="DisplayedScalarRangeModeComboBox">
        <property name="toolTip">
         <string>Set the scalar range mode flag. Data (auto) updates the Display Scalar Range from the scalar range of the data.</string>
        </property>
        <item>
         <property name="text">
          <string>Active scalar (auto)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Color table (LUT)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data type</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manual</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="DisplayedScalarRangeLabel">
        <property name="text">
         <string>Displayed Range:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="qMRMLRangeWidget" name="DisplayedScalarRangeWidget">
        <property name="toolTip">
         <string>Set the display scalar range, used when the Display option is selected.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="DisplayedScalarRangeLabel">
        <property name="text">
         <string>Threshold:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="ThresholdCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Apply thresholding to your displayed model based on its scalar values. Some cells normals might be flipped, in which case you
should set &quot;backface&quot; and &quot;frontface&quot; to OFF in the Representation options to visualize all cells.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="ctkRangeWidget" name="ThresholdRangeWidget" native="true">
        <property name="toolTip">
         <string>Range used to threshold scalar values.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLDisplayNodeWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLDisplayNodeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkVTKDataSetArrayComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkVTKDataSetArrayComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLRangeWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
